task_03.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Task 03: mortgage calculator to calculate the
+lifetime compound interest of a loan."""
+
+
+import decimal
+
+NAME = raw_input("What is your name? ")
+PRINCIPAL = int(raw_input(
+    "What is the amount of your principal (the amount being borrowed)? "))
+DURATION = int(raw_input("For how many years is this loan being borrowed? "))
+PREQUALIFIED = raw_input("Are you pre-qualified for this loan? ")
+
+
+if PRINCIPAL > 0 and PRINCIPAL < 200000:
```

Comment:
You did a good job building the conditions tree to set the rates, however, you didn't account for _every possible combination of principal, duration, or prequalification status. So what's missing here is a default value for `RATE` before you begin the conditions at L16 eg, `RATE = None`